src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java

```diff
@@ -23,7 +22,7 @@
 
 /**
  * Extension point to subscribe events from GH, which plugin interested in.
- * This point should return true in {@link #isApplicable(AbstractProject)}
+ * This point should return true in {@link #isApplicable}
```

Comment:
That was my point - I thought you needed to add `(argumentType)` for links to methods. 
I was mistaken.